### ISSUE TITLE
import show_prediction issue

### ISSUE THREAD

**Issue by akuuzii** [TeamHG-Memex/eli5#405]
from eli5 import explain_weights
from eli5 import show_prediction
Traceback (most recent call last):
  File "<input>", line 1, in <module>
ImportError: cannot import name 'show_prediction' from 'eli5' (C:\randomdirectory\PycharmProjects\cardiowebML\venv\lib\site-packages\eli5\__init__.py)

for some reason, I can import some functions but not show_predictions

**Comment by teabolt** [comment]
Hi @akuuzii. Can you show us the output of `pip freeze` when using your venv? Does `from eli5 import explain_prediction` work? For your reference `show_prediction` is implemented at https://github.com/TeamHG-Memex/eli5/blob/master/eli5/ipython.py#L306

**Comment by akuuzii** [comment]
attrs==20.3.0
eli5==0.11.0
et-xmlfile==1.0.1
graphviz==0.16
Jinja2==2.11.3
joblib==1.0.1
MarkupSafe==1.1.1
numpy==1.20.2
openpyxl==3.0.7
pandas==1.2.3
python-dateutil==2.8.1
pytz==2021.1
scikit-learn==0.24.1
scipy==1.6.2
six==1.15.0
sklearn==0.0
tabulate==0.8.9
threadpoolctl==2.1.0
xgboost==1.3.3

**Comment by akuuzii** [comment]
I am able to import explain_prediction.
